deploy/examples/jbpm-quarkus-example-binary-builds.yaml

```diff
@@ -3,7 +3,5 @@ kind: KogitoApp
 metadata:
   name: example-quarkus-flash
 spec:
-  # you only need to inform the correct image tag if you wish to build from a custom image
-  build:
-    imageVersion: 0.9.1 # will use it as a reference to create the imagestreams that the Operator uses
+  build: {}
```

Comment:
IMHO this can be removed, keeping just `spec{}`

Reply:
We can't, there's a YAML validation test that enforces this